app/page/schedule/List/List.js

```diff
@@ -10,34 +10,73 @@ define([
 ], function (Page, can, content, Schedule) {
 	return Page.extend(
 		{
-
+			defaults: {
+				
+				template: content
+			}
 		},
 		{
-			template : content,
 
-			init: function (element) {
-                var isLoading = can.compute(true);
+			init: function (element, options) {
+
+				this.modelView = new can.Map(
+					{
+						schedule:null,
+						isLoading: true
+					}
+				);
+
+				element.html(options.template(this.modelView));
+
+				this.setSchedules();
+
+            },
 
-                var promise = Schedule.findAll({
+            setSchedules: function () {
+            	var promise = Schedule.findAll({
 	                /**
 	                 * Saljemo na server _expand parametar kako bismo dobili i objekat menija u samom schedule objektu
 	                 */
 	                '_expand' : 'menu'
-                });
+                }); 
 
-                promise.always(
+                var that = this;
+
+				promise.always(
                     function () {
-                        isLoading(false);
+                        that.modelView.attr('isLoading', false);
                     }
                 );
+				this.modelView.attr('schedule', new Schedule.List(promise));
+
+				/**
+				 * Omogucavamo uvezivanje metoda
+				 */
+				return this;
+			},
+
+            removeSchedule: function(link, event){
+     
+            	event.preventDefault();
+            	var that = this;
```

Comment:
@jeffsouln `that` ili `self` je ok za koristiti ukoliko imas vise izraza u kojima promenljiva ucestvuje. U ovom slucaju je bilo jednostavnije uraditi `.bind(this)` na samoj funkciji. 

Takodje praksa je da deklaracija promeljivih bude na vrhu. 
http://www.sitepoint.com/demystifying-javascript-variable-scope-hoisting/